.github/workflows/linting.yml

```
@@ -5,11 +5,21 @@ jobs:
   prose:
     runs-on: ubuntu-latest
     steps:
+    # Checkout this repo
     - name: Checkout
       uses: actions/checkout@master
```

```suggestion
      uses: actions/checkout@v2
      with:
        ref: ${{ github.event.pull_request.head.sha }}
        fetch-depth: 1
```
- do not fetch history
- get the current branch only